submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/SecurityProperties.kt

```diff
@@ -14,10 +14,21 @@ class SecurityProperties {
 
     @NestedConfigurationProperty
     var instanceKeys = InstanceKeys()
+
+    override fun toString(): String {
```

Comment:
We can transform into data class as spring support constructor binding

Reply:
Taking a look